ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_flow_tracking_link.schema.yml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+$defs:
+  flow_tracking_link:
+    type: dict
+    keys:
+      enabled:
+        type: bool
+      type:
+        type: str
+        description: Flow tracker type.
+        valid_values:
+        - sampled
+        - hardware
```

Comment:
are we sure we can never configure `sampled` and `hardware` on the same interface? When using `FastCli -G` on a cloudeos lab I get:

```
interface Ethernet1
   description ATT_Colt55
   no switchport
   flow tracker hardware TOTO
   flow tracker sampled BLAH
```

but it may be only because I am unlocking the CLI (only hardware is supported otherwise)